komodo/release_transpiler.py

```diff
@@ -112,6 +180,15 @@ def main():
         required=True,
         help="File containing explicit matrix packages (default: None)",
     )
+    matrix_parser.add_argument(
```

Comment:
I thought the Python version (and any arbitrary matrix 'coordinates') could be encoded in the `coords` argument?

Reply:
As the code of the function `combine` is of now, it needs only the python versions.
I guess this is due to historical reasons.
The options were:
1. keep `PY_VERSION=[3.8]` hardcoded as it was previously;
2. pass `coords` even though `rhel_ver` is not used, which means we are passing an unnecessary argument;
3. let only `--py-coords` as an optional argument;
4. remove the function `combine` as a whole - it is not used by us now. I preferred not because it maybe an useful feature for users outside scout or maybe can be useful in the future.

I preferred the third option, but would you prefer the second one?

Reply:
I think we should avoid adding Equinor-specific arguments so I think it should accept the same matrix definition (i.e. `coords` --- which should maybe be called `matrix`).

If we don't want to refactor it now because we don't use it, maybe we should deprecate it.

Reply:
Is it possible to avoid adding another specific 'coordinate' argument here?

Reply:
> Is it possible to avoid adding another specific 'coordinate' argument here?

I understand you refer to 'coordinate' to `--py_coords` argument. If so, it is an argument keep the previous functionaty of `combine`. But it is optional and by default the versions to be combined will be retrieved in the `release_folder`.

As it is an optional argument, I don't see the problem of the solution. Do you have something particularly in mind?
Remove the option and let only the default behaviour?

Reply:
If there's really no way to avoid adding an argument to the interface that we will have to deprecate when we eventually fix all this Py/RHEL stuff, then it has to be done. I trust that you've considered all the alternatives.